csharp/extractor/Semmle.Extraction.CSharp/Entities/Expressions/Lambda.cs

```diff
@@ -24,6 +24,18 @@ private void VisitParameter(ParameterSyntax p)
         private Lambda(ExpressionNodeInfo info, CSharpSyntaxNode body, IEnumerable<ParameterSyntax> @params)
             : base(info)
         {
+            var symbol = cx.GetModel(info.Node).GetSymbolInfo(info.Node).Symbol as IMethodSymbol;
+
+            if (symbol is object)
+            {
+                Modifier.ExtractStaticModifier(cx, info.Context.TrapWriter.Writer, this, symbol);
+                Modifier.ExtractAsyncModifier(cx, info.Context.TrapWriter.Writer, this, symbol);
```

Comment:
I would just call `ExtractModifiers`, in case new modifiers are allowed later.

Reply:
I pushed an extra commit to do this. (It does extract more modifiers (`private`) this way.)